care/facility/summarisation/tests_summary.py

```diff
@@ -0,0 +1,104 @@
+from care.facility.models import FacilityRelatedSummary, Facility, PatientSample
+
+from celery.decorators import periodic_task
+from celery.schedules import crontab
```

Comment:
Codacy found an issue: [third party import "from celery.schedules import crontab" should be placed before "from care.facility.models import FacilityRelatedSummary, Facility, PatientSample"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5838242)